### PR TITLE
:bricks: Conform to libhal

### LINKED ISSUE #3: Conform to libhal `device` driver library
Setup the repository to met the requirements of a `device` driver library.

### PR DESCRIPTION
**Pull request by MaliaLabor**
Resolves #3

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+class LibmpuConan(ConanFile):
+    name = "libmpu"
+    version = "0.0.1"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libhal.github.io/libmpu"
+    description = (
+        "A collection of drivers for the MPU series IMU devices")
+    topics = ("servo", "smart servo", "PID")
```
Comment (kammce) [inline conanfile.py]: Change this to reflect the acceleration and motion characteristics of the driver.

**Review by kammce — CHANGES_REQUESTED** [review]
Also lets get the unit tests to work even if they are empty.

**Comment by kammce** [comment]
To get the unit tests to work, we have to make run.sh executable using `chmod +x ./tests/run.sh`

**Review by kammce — APPROVED** [review]

**Comment by kammce** [comment]
Will merge with tests breaking.
